### PR TITLE
Iosxe/show stackwise virtual dual active detection

### PR DESCRIPTION
**Pull request by reustaquio22**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+
+# =================================================
+# Schema for:
+#  * 'show_stackwise_virtual_dual_active_detection'
+# =================================================
+class Show_Stackwise_Virtual_Dual_Active_DetectionSchema(MetaParser):
+    """Schema for show_stackwise_virtual_dual_active_detection."""
+
+    schema = {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py:14]: better to have the key `dad_port`. 
```
    schema = {
        "switches": {
            int: {
                "dad_port": {
                    str: {
                        "status": str
                    }
                }
            }
        }
    }

File: src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                if clean_line_strip.startswith(remove_lines):
+                    clean_lines.remove(clean_line)
+            return clean_lines
+
+        remove_lines = ('----', 'Switch', 'Dual')
+        out = filter_lines(raw_output=out, remove_lines=remove_lines)
+        switch_id = ''
+
+        for line in out:
+            match = switch_id_capture.match(line)
```
Comment (ahersi2-cisco) [inline src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py:75]: Please add a matching line as a comment above this line

File: src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            if not dad_dict.get('switches'):
+                dad_dict['switches'] = {}
+            if match:
+                groups = match.groupdict()
+                switch_id = int(groups['switch_id'])
+                dad_dict['switches'].update({switch_id: {groups['dad_port']: {'status': groups['status']}}})
+            port_match = port_capture.match(line)
```
Comment (ahersi2-cisco) [inline src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py:82]: Same here, please add a matching line as a comment above

File: src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py (hunk 1 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py]: `dad_port` is now the root key

File: src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+
+# =================================================
+# Parser for:
+#  * 'show_stackwise_virtual_dual_active_detection'
+# =================================================
+class Show_Stackwise_Virtual_Dual_Active_Detection(Show_Stackwise_Virtual_Dual_Active_DetectionSchema):
+    """Parser for show stackwise-virtual dual-active-detection"""
+
+    cli_command = ['show stackwise-virtual dual-active-detection']
```
[the remark this answers is not in the document]
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py]: This is changed to a string

File: src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py (hunk 3 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py]: The match line has been added

File: src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py (hunk 2 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_stackwise_virtual_dual_active_detection.py]: This has been updated with the matching line

**Comment by ahersi2-cisco** [comment]
Hi,
Can you please resolve conflicts. Thank you.

**Pull request merged**